### ISSUE TITLE
ReferenceError: internalBinding is not defined

### ISSUE THREAD

**Issue by bonesoul** [andrewrk/node-s3-client#215]
tracked it down to here.

anyone also having the issue?

**Comment by JohanBengtsson** [comment]
@bonesoul  Yes, I think it's due to the dependency to an old version of `graceful-js` but I'm not 100% sure. I'm investigating it right now.

**Comment by bonesoul** [comment]
graceful-js uses the natives package i guess.

i think it's the offending one.

https://github.com/nodejs/node/issues/25132

**Comment by JohanBengtsson** [comment]
@bonesoul  I fixed the issue by resolving `grafeul-js` as one of the comments suggested (we're using yarn) 

```
"resolutions": {
    "graceful-fs": "4.1.15"
}
```

**Comment by bonesoul** [comment]
is there a way to make the resolution fixed with npm?

**Comment by JohanBengtsson** [comment]
not sure,  don't think NPM has it natively but this package seems to give the same functionality https://www.npmjs.com/package/npm-force-resolutions ? I guess you'll need to start using `package-lock.json` if you don't already @bonesoul

**Comment by bonesoul** [comment]
i already use package-lock and will give a try to npm force resolutions.
